.github/workflows/publish-rust.yml

```diff
@@ -88,7 +88,9 @@ jobs:
           cargo-cache-fallback-key: cargo-publish
 
       - name: Install Cargo Release
-        run: which cargo-release || cargo install cargo-release
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-release
```

Comment:
In other repos we have the version fixed (e.g., [pinocchio](https://github.com/anza-xyz/pinocchio/blob/main/.github/actions/setup/action.yml#L86)) since the rust requirement might change in patch versions: version `0.25.17` requires rust 1.80 while `0.25.18` (latest) requires 1.85.

Perhaps it would be better to fix it to `cargo-release@0.25.17` since I think the active rust version is `1.84.1`.